frontend/src/app/app-routing.module.ts

```diff
@@ -443,16 +444,42 @@ export const routes: Routes = [
                     component: T4CSettingsComponent,
                   },
                   {
-                    path: 'create',
-                    data: { breadcrumb: 'new' },
+                    path: 'create-instance',
+                    data: { breadcrumb: 'New Instance' },
                     component: EditT4CInstanceComponent,
                   },
                   {
-                    path: 'instance/:instance',
+                    path: 'instance',
                     data: {
-                      breadcrumb: (data: Data) => data.t4cInstance?.name,
+                      breadcrumb: 'Instances',
                     },
-                    component: EditT4CInstanceComponent,
+                    children: [
+                      {
+                        path: ':instance',
+                        data: {
+                          breadcrumb: (data: Data) => data.t4cInstance?.name,
+                        },
+                        component: EditT4CInstanceComponent,
+                      },
+                    ],
```

Comment:
This creates "Instances" breadcrumbs that lead to nowhere when clicked. Same for License Servers below.

Reply:
Noticed the same and is already fixed by specifying a custom redirect.